crates/bevy_ui/src/focus.rs

```diff
@@ -52,6 +53,25 @@ impl Default for Interaction {
     }
 }
 
+/// A component storing the position of the mouse relative to the node, (0., 0.) being the upper-left corner and (1., 1.) being the bottom-right
+/// I can be used alongside interaction to get the position of the press.
+/// If the mouse is not over the node, the value will go beyond the range of (0., 0.) to (1., 1.)
+#[derive(
+    Component,
+    Deref,
+    DerefMut,
+    Copy,
+    Clone,
+    Default,
+    PartialEq,
+    Debug,
+    Reflect,
+    Serialize,
+    Deserialize,
+)]
+#[reflect(Component, Serialize, Deserialize, PartialEq)]
+pub struct RelativeCursorPosition(pub Vec2);
```

Comment:
I would like a helper method that tells you whether or not you're over the node.